pkg/manifests/manifests_test.go

```diff
@@ -2802,6 +2811,40 @@ func TestNodeExporter(t *testing.T) {
 	}
 }
 
+func TestNodeExporterCollectorSettings(t *testing.T) {
```

Comment:
nit: shouldn't we also test the default behaviour, this is when `cpufreq` is disabled we expect to see `--no-collector.cpufreq`?

Reply:
+1 
Since we got rid of `unsetArg` a second test case ensuring that either `--no-collector.cpufreq` or `--collector.cpufreq` make sense.

Reply:
Thanks for the feedback, will add a second test :)